helm-charts/interoperator/conf/settings.yaml

```
@@ -14,6 +14,8 @@ defaults: &defaults
   sys_log_level: info
   enable_circuit_breaker: true
   http_timeout: 175000
+  allowConcurrentOperations: false
+  allowConcurrentBindingOperations: false
```

```suggestion
  allowConcurrentBindingOperations: true
```
Since there are higher chances of concurrent creation of bindings for the service instance in Kubernetes landscapes, I feel it might be good to keep this default as `true`.

---

Now, reading these configs from values instead of hardcoding. Default value for both configs has been kept `true` to maintain status quo. We will communicate to service owners along with release about these configs and let them decide about it. Thanks @anoopjb for noticing this!